src/observable/from.ts

```diff
@@ -15,19 +17,30 @@ export class FromObservable<T> extends Observable<T> {
     super(null);
   }
 
-  static create<T>(ish: any, scheduler: Scheduler = null): Observable<T> {
+  static create<T>(ish: any, mapFnOrScheduler: Scheduler | ((x: number, y: any) => T), thisArg?: any, lastScheduler?: Scheduler): Observable<T> {
+    let scheduler: Scheduler;
+    let mapFn: (x: number, y: any) => T;
+    if (typeof mapFnOrScheduler === 'function') {
```

Comment:
You can use [isFunction()](https://github.com/ReactiveX/RxJS/blob/master/src/util/isFunction.ts) here.